lib/geordi/db_cleaner.rb

```diff
@@ -5,12 +5,21 @@
 module Geordi
   class DBCleaner
 
-    def initialize(extra_flags)
-      puts 'Please enter your sudo password if asked, for db operations as system users'
-      puts "We're going to run `sudo -u postgres psql` for PostgreSQL"
-      puts '               and `sudo mysql`            for MariaDB (which uses PAM auth)'
-      `sudo true`
-      Interaction.fail 'sudo access is required for database operations as database users' if $? != 0
+    def initialize(extra_flags, system_user: false)
+      @system_user = system_user
+
+      if @system_user
+        puts 'Please enter your sudo password if asked, for db operations as system users'
+        puts "We're going to run `sudo -u postgres psql` for PostgreSQL"
+        puts '               and `sudo mysql`            for MariaDB (which uses PAM auth)'
+        `sudo true`
+        Interaction.fail 'sudo access is required for database operations as database users' if $? != 0
+      else
+        puts "We're going to run `psql` for PostgreSQL"
+        puts '               and `mysql`            for MariaDB'
+        puts 'That means, we are assuming your local user has the required permissions to drop the selected databases'
```

Comment:
The "we're going to run" part is to assure the user he can entrust us with their sudo password. I'd leave that away in this case and just state the assumption. However, we cannot mention the command option here since we do not know which command will call us. We'll need to do this inside the command, so we can drop the `else` part here.